cibuild.cmd

```diff
@@ -145,5 +147,15 @@ exit /b 1
 @REM Kill any instances of msbuild.exe to ensure that we never reuse nodes (e.g. if a non-roslyn CI run
 @REM left some floating around).
 
-taskkill /F /IM vbcscompiler.exe 2> nul
-taskkill /F /IM msbuild.exe 2> nul
+@REM An error-level of 1 means that the process was found, but could not be killed.
+taskkill /F /IM vbcscompiler.exe > nul
```

Comment:
Consider the case of multiple users on the same machine.  How will taskkill react if a different user is running msbuild / vbcscompiler?  Will it attempt to kill the process and issue an error code?